packages/fast-components-react-base/src/select/select.tsx

```diff
@@ -421,7 +467,7 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, Select
         }
         const isItemSelected: boolean = state.selectedItemIndex !== 0;
         return (
-            <button
+            <Button
```

Comment:
Is this related to your change or a different bug? If it's not directly related can we pull it out and tackle it elsewhere?

Reply:
sure